### ISSUE TITLE
Missing image for Jamaal Bowman B001223

### ISSUE THREAD

**Issue by pcraciunoiu** [unitedstates/images#203]
Noticed that Jamaal Bowman doesn't have a picture. I didn't check for others yet.

**Comment by pcraciunoiu** [comment]
Could pull it from Wikipedia maybe? https://en.wikipedia.org/wiki/Jamaal_Bowman#/media/File:Jamaal_Bowman_117th_U.S_Congress.jpg

**Comment by Andrew-Chen-Wang** [comment]
@pcraciunoiu I believe the entire 117th congress isn't even here due to our source being from [Government Printing Office's Member Guide](https://memberguide.gpo.gov) which as of today doesn't have the current in session Congress.

For future users, I'll be making a separate repo in the [Hear-Ye](https://github.com/Hear-Ye) to get this done for my personal needs to make this into a package. For @JoshData or whoever is maintaining this repository, I highly recommend using this as the new scraper source: https://www.congress.gov/search?q={%22congress%22:[%22117%22],%22source%22:[%22members%22]}&pageSize=250

Edit: seems like all the photos are stored by bio guide id with this format: https://www.congress.gov/img/member/c001051.jpg The above source has a couple people missing, but it's better than the gpo or below source

or (this source seems to not have images as up to date) https://bioguide.congress.gov/search?index=%22bioguideprofiles%22&size=96&matches=[]&filters={%22jobPositions.congressAffiliation.congress.name%22:[%22The%20117th%20United%20States%20Congress%22]}&sort=[{%22_score%22:true},{%22field%22:%22familyName%22,%22order%22:%22asc%22},{%22field%22:%22middleName%22,%22order%22:%22asc%22},{%22field%22:%22givenName%22,%22order%22:%22asc%22}]

also for the maintainers, highly recommend using GitHub actions' cron job (or circle ci as for your other repository) to autoupdate the repository. take a look at https://github.com/Hear-Ye/congress-data (ref that's why I needed a package: for maintainability via versions)

**Comment by pcraciunoiu** [comment]
@Andrew-Chen-Wang thanks for this. I'll take a look at using congress.gov directly - though if you end up releasing a package, please let me know / post here?

**Comment by hugovk** [comment]
Would welcome PRs to update the scraper here to a better source, as long as they're public domain, but I think the images at https://www.congress.gov/img/member/c001051.jpg are a little small.

GitHub Actions cron is a good idea, although it seems the member guide is updated very rarely.

**Comment by Andrew-Chen-Wang** [comment]
@hugovk will definitely PR here as well if the images are actually bigger; the repos here aren't very well maintained, and even if I was given some perms in this repo, I would feel the need to constantly ask for reviews to make sure things are gtg; that's why I'm mostly porting everything to my own repositories for use.

> https://www.congress.gov/img/member/c001051.jpg are a little small

I'm not sure where else to get the 117th congressmen's images. I'm developing a mobile application, so it doesn't matter to me. Please let me know if you find a better source @hugovk !

<details><summary>Also additionally, this is the reply I got about the congress.gov images' copyright. Turns out they're from bioguide.congress.gov, but that doesn't make too much sense since there are more photos on congress.gov than the bio guide search. Note: I guess I asked this question to the wrong person</summary>

The Library of Congress does not grant or deny permission to use the content mounted on its website. That being said, the photos on Congress.gov come from the online edition of the "Biographical Directory of the United States Congress". https://bioguide.congress.gov/

A copyright statement regarding the images found in the "Biographical Directory of the United States Congress" can be found at: https://bioguide.congress.gov/copyright 

If you have any additional questions about the photos published in the "Biographical Directory of the United States Congress", please use the Senate and House contact information provided on this copyright statement page.

Thank you for using the Library's Ask-A-Librarian service.

Sincerely,

The Researcher and Reference Services Division/KD
Library of Congress

---

In other words, I'll also call to get some more clarifying information.

After reading through everything a couple of times, the links I've provided which all use the bio guide website don't really guarantee CC0 compatibility.

</details>

**Comment by mdrayer** [comment]
Regarding the original issue, the images are now available:
- https://theunitedstates.io/images/congress/original/B001223.jpg
- https://theunitedstates.io/images/congress/450x550/B001223.jpg
- https://theunitedstates.io/images/congress/225x275/B001223.jpg

However, the associated metadata file is missing: https://github.com/unitedstates/images/blob/gh-pages/congress/metadata/B001223.yaml
